unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalInputEventResultsExtensions.cs

```diff
@@ -6,28 +6,20 @@ namespace DCL.ECS7.InternalComponents
 {
     public static class InternalInputEventResultsExtensions
     {
-        internal const int MAX_AMOUNT_OF_POINTER_EVENTS_SENT = 30;
-
         public static void AddEvent(this IInternalECSComponent<InternalInputEventResults> component, IParcelScene scene,
             InternalInputEventResults.EventData data)
         {
             var model = component.GetFor(scene, SpecialEntityId.SCENE_ROOT_ENTITY)?.model ??
                         new InternalInputEventResults()
                         {
                             lastTimestamp = 0,
-                            events = new Queue<InternalInputEventResults.EventData>(MAX_AMOUNT_OF_POINTER_EVENTS_SENT)
+                            events = new Queue<InternalInputEventResults.EventData>(6)
```

Comment:
This magic number should be in a constant or explained.

Reply:
I removed the `6` because it isn't necessary :)